clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2009,6 +2009,16 @@ device kernel, the attribute is ignored and it is not propagated to a kernel.
   }];
 }
 
+def SYCLIntelUsesGlobalWorkOffsetDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "uses_global_work_offset (IntelFPGA)";
+  let Content = [{
+Applies to a device function/lambda function or function call operator (of a
+function object). If 0, ``get_global_offset(X)`` may not be called by the
+kernel. Valid values are 0 and 1.
```

Comment:
Should the documentation specify that non-1 positive values are still accepted as `true`?

Reply:
> let Args = [BoolArgument<"Enabled">];

Reply:
I mean, even they arg > 1 is acceptable - yet let's not teach users bad manners.